src/spatial/explog.hpp

```diff
@@ -168,7 +169,7 @@ namespace se3
     Scalar n2 = r.squaredNorm(),a,b,c;
     Scalar n = math::sqrt(n2);
     
-    if (n < 1e-6)
+    if (n < TaylorSeriesExpansion<Scalar>::template precision<4>())
     {
 
       a =   Scalar(1)           - n/Scalar(6)   + n2/Scalar(120);
```

Comment:
Btw, aren't this Taylor expansion is wrong ? I think they should be:
```cpp
a =   Scalar(1)           - n2/Scalar(6);
b = - Scalar(1)/Scalar(2) - n2/Scalar(24);
c =   Scalar(1)/Scalar(6) - n2/Scalar(120);
```

Reply:
You're right too. I will apply this change.